fic_modules/helper_functions.py

```diff
@@ -1,10 +1,13 @@
+"""
+This module contains functions that are for other scopes wich aim to get better
+ functionality out of the code and not directly related to the script.
+"""
 import re
 import sys
 from github import GithubException
 from fic_modules.configuration import (
     REPOSITORIES,
     REPO_LIST,
-    TOKEN,
```

Comment:
Do we not use this anymore?

Reply:
We are using the token just for the creation of the github object, that we are doing it in the configuration file. There is no need to import the token, since we just import the github object when we need it.
That is done in L17 in the configuration.py (`GIT = Github(TOKEN)`)